### DISCUSSION TITLE
Example File Builder

### DISCUSSION THREAD

**Discussion by miniscruff** [caarlos0/env#267] [Show and tell]
Hey again Carlos, I wanted to share a little project I built that generates an example file from a configuration struct that env would parse into. The idea being you would commit the example file, create an `.env` file locally via something like `cp .env.example .env` and have a starting point of values to work from. Sort of like how other major projects give you a default config with comments and things, for example nginx. 

Project: https://github.com/miniscruff/envexample

Oh, it is also using goreleaser of course :smirk:

**Comment by caarlos0** [comment]
Awesome!

Thanks for sharing 🤘
